inspirehep/modules/workflows/workflows/article.py

```diff
@@ -278,15 +278,6 @@
 
 
 HALT_FOR_APPROVAL_IF_NEW_OR_STOP_IF_NOT_RELEVANT = [
-    IF_NOT(
-        is_record_relevant,
-        [
-            reject_record('Article automatically rejected'),
-            mark('approved', False),
-            save_workflow,
-            stop_processing,
-        ],
-    ),
     preserve_root,
```

Comment:
Do we need to preserve roots of rejected records?

Reply:
Good point, moving it after the record is eventually accepted.

Reply:
We'll do it later because moving the `preserve_root` requires the migration of workflows after. Let'$ deploy this PR in prod first, and we'll work on this after.